content/tagging/using_tags.md

```diff
@@ -12,105 +12,271 @@ further_reading:
   text: Learn how to assign tags
 ---
 
-After you have assigned tags at the host and [integration][1] level, you can start using them to filter and group in interesting ways. There are several places you can use tags:
+After [assigning tags][1], you can start using them to filter and group in interesting ways.
 
-- [Events List][2]
-- [Dashboards][3]
-- [Infrastructure List][4]
-- [Host Map][5]
-- [Monitors][6]
+## Events
 
-## Using tags in the Events List
+The [Events Stream][2] shows all the events that have occurred in your environment over the time period specified. Using tags to filter down this list is very beneficial. Enter `tags:` followed by a tag to see all the events that come from a host or [integration][3] with that tag. The example below is used to search for the tag `service:coffee-house` using `tags:service:coffee-house`. To search multiple tags, separate each tag by a comma - `tags:service:coffee-house,host:coffeehouseprod`.
 
-The [Events List][2] shows you all the events that have occurred in your environment over the time period specified. This can be overwhelming so you can use tags to filter down the list based on the tags you have assigned. You can enter any text you want in the search box above the Event List and a full text search is performed. You can also enter `tags:` followed by a tag to see all the events that come from a host or [integration][1] with that tag. The example in the image is the tag `role:cassandra`. So the search text is `tags:role:cassandra`.
+{{< img src="tagging/using_tags/eventtags.png" alt="Events List and Tags" responsive="true" style="width:80%;">}}
 
-{{< img src="tagging/eventtags.png" alt="Events List and Tags" responsive="true" style="width:70%;">}}
+## Dashboards
 
-## Using tags in Dashboards
+{{< tabs >}}
+{{% tab "Assignment" %}}
 
-You can use tags to narrow down the metrics to display on a [dashboard graph][7], or to create groups of metrics to display.
-To narrow down the metrics to display, enter the tag in the `from:` textbox.
+You can use tags to narrow down the metrics to display on a [dashboard graph][4], or to create groups of metrics to display.
```

Comment:
let's remove `you can` 😃 and be consistent with the vocabulary choice here:

`You can use tags to narrow down the metrics to display on a [dashboard graph][4], or to create groups of metrics to display.`

`Use tags to filter metrics to display in a [dashboard graph][4], or to create aggregated groups of metrics to display.`